tests/spread/spread_01.cpp

```diff
@@ -376,17 +376,11 @@ main(int argc, char** argv)
             half_f_vector.close();
         }
 
-        std::ostringstream out;
+        // the partitioning isn't relevant when we only have one processor
         if (IBTK_MPI::getNodes() != 1)
         {
-            // partitioning is only relevant when there are multiple processors
-            Pointer<PatchLevel<NDIM> > patch_level =
-                patch_hierarchy->getPatchLevel(patch_hierarchy->getFinestLevelNumber());
-            const BoxArray<NDIM> boxes = patch_level->getBoxes();
```

Comment:
Here is the mistake - @boyceg pointed out that this gets all boxes (i.e., across all processors). Fortunately a second bug here (printing boxes to plog when I intended to print to out) mostly canceled out the first bug, so we print the correct boxes in the correct order but are just missing the labels.